src/app/perspective-status.component.ts

```diff
@@ -385,27 +386,65 @@ export class PerspectiveStatus implements OnChanges, AfterViewInit, AfterViewChe
   }
 
   /**
-   * Updates the gradient color function for the shape based on the
+   * Updates the gradient color scale for the shape based on the
    * scoreThresholds.
    */
   updateGradient() {
```

Comment:
This is getting kind of involved... worth a test?

Reply:
Yeah, I agree. I'm going to be adding tests for a few different things--I will add this to the list. I'm going to make a bug for it, actually, to keep track of things we need tests for.